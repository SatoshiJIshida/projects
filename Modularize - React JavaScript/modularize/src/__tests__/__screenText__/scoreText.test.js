import React from "react";
import { shallow } from "enzyme";
import ScoreText from "../../components/screenText/scoreText";

/**
 * Library Jest taken from source https://jestjs.io/
 * Library Enzyme taken from source https://www.npmjs.com/package/enzyme
 */

describe("Score Text", () => {
  it("should be defined", () => {
    expect(ScoreText).toBeDefined();
  });

  it("should render correctly", () => {
    const scoreText = shallow(<ScoreText name="scoreText-test" />);
    expect(scoreText).toMatchSnapshot();
  });
});
