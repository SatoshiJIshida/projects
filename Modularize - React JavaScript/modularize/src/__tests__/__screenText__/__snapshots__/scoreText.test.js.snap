// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score Text should render correctly 1`] = `ShallowWrapper {}`;
